app/src/main/java/no/usn/mob3000/domain/usecase/social/userProfile/FetchUserByIdUseCase.kt

```diff
@@ -0,0 +1,19 @@
+package no.usn.mob3000.domain.usecase.social.userProfile
+
+import no.usn.mob3000.data.repository.social.UserRepository
+import no.usn.mob3000.domain.model.auth.UserProfile
+/**
+ *
+ * This use case retrieves the user's profile information.
+ * @param userRepository The repository for user operations.
+ *
+ * @author Husseinabdulameer11
+ * @created: 2024-11-10
+ */
+class FetchUserByIdUseCase(

```

Comment:
There were two comments here incorrectly marked as resolved, but I've deleted them to make it more obvious what the issue is since it was only made worse.

1. KDocs should not be touching the import declaration. Spacing.
2. There is an empty starting line in the KDoc. Spacing.
3. The descriptive text and the KDoc property `@param` should be separated by an empty KDoc line. Spacing.
4. The `@created` KDoc property should not have a colon. It should be `<property> <value>`.